packages/client/src/styles/globals.css

```
@@ -980,14 +980,67 @@
+ * mouse drag into scrollLeft and sets data-fade-left / data-fade-right from the live scroll
+ * position, and the fade is applied as a mask rather than a solid gradient overlay: the row
+ * renders over the chat background image, the floating Mari panel, and the conversation input
+ * alike, so there is no single base color an overlay could blend into. Both stops stay 0 and
+ * the mask rule below does not match at all while nothing is clipped, which leaves the
+ * wrapping desktop layout untouched.
+ */
 .mari-suggestion-chips {
+  --mari-suggestion-chips-fade: 1.5rem;
+  --mari-suggestion-chips-fade-left: 0px;
+  --mari-suggestion-chips-fade-right: 0px;
   display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)